react/formatIOMessage.ts

```diff
@@ -9,7 +10,7 @@ type AdaptedMessageDescriptor = MessageDescriptor & {
   intl: IntlShape
 }
 
-export type Values = Record<string, string | number>
+export type Values = Record<string, string | number | ReactElement>
```

Comment:
Does react-intl export this type? Maybe we should their type instead.

Reply:
It doesn't... 
![image](https://user-images.githubusercontent.com/12702016/102619117-b06fbc80-411a-11eb-902c-7ffceb797ec8.png)

Reply:
That would be awesome! But they don't expose it :(